fischkapp/src/components/cards/BackCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { CardsWrapper } from "../styles/SitesCards.styled";
+import { StyledInput } from "../styles/SitesCards.styled";
+import { StyledCardButton } from "../styles/buttons/CardButton";
+import { StyledButtonsContainer } from "../styles/SitesCards.styled";
+import { StyledControlCardPanel } from "../styles/SitesCards.styled";
+import { StyledDeleteButton } from "../styles/buttons/DeleteButton";
+import deleteIcon from "../../assets/deleteIcon.svg";
+
+type BackCardProps = {
+  onFlipCard: Function;
```

Comment:
**nitpick:** We should type functions explicitly - in this case it should be: `onFlipCard: () => void`